### ISSUE TITLE
Native module not found...

### ISSUE THREAD

**Issue by nickich** [jaredallard/nexe#199]
I'm using webrtc-native module in my app, which can't be found after nexe build the executable.

```
node.js:866
      throw new Error('No such native module ' + id);
      ^

Error: No such native module webrtc-native
```

The module is 'require' as:
```
var webrtcvar = "webrtc-native";
var WebRTC = require(webrtcvar);
```

How I'm supposed to add a native module?

**Comment by jaredallard** [comment]
We don't support native modules as of right now, I'd try using `nexe@0.4.7` and see if it throws the same error, as this may be actually due to `browserify` and not it being a native module.

We also don't support dynamic requirements.

**Comment by BrighTide** [comment]
nexe@0.4.7 is not a valid install?

**Comment by nickich** [comment]
I can't find nexe@0.4.7.
From the log I see a jump from 0.4.2 to 1.0.0:
-  "version": "0.4.2",
+  "version": "1.0.0",

committed on Jan 22

From the doc:
"Use the techniques below for working around dynamic require statements to exclude the module from the bundling, and deploy along side the executable in a node_module folder so your app can find it. Note: On windows you may need to have your app be named node.exe if .node file depends on node."
That's what I try to use to get it working. Unfortunately the executable can't find the module.

**Comment by jaredallard** [comment]
That not longer applies to 1.x.x unfortunately. So if it doesn't work with 0.4.2 then I'm not sure how to help  at the moment. Issues with browserify are apparent and I'm trying my best to find work arounds with limited time at the moment!

**Comment by nickich** [comment]
Someone suggested to use
GLOBAL.require('./mymodule.node');
which seems to work in this case. Need more testing, but looks promising.

**Comment by synaestheory** [comment]
I just wanted to take a moment to thank you for all of your hard work on this project @jaredallard.  I'm digging through right now trying to find a solution that seems to be related to this native module issue.  aws-sdk is trying to require a `metadata.json` (though I'm not sure if it is a dynamic require just yet) and so my app won't run after compiling.  Thanks again for the great work.  I appreciate all of the time you've sunk into getting this project where it is and continuing to maintain it.

**Comment by jaredallard** [comment]
Fixed in v2.0.0. So, closing.

**Comment by adifire** [comment]
Hey, thanks for working on the nexe, it's great! I was wondering if there is a workaround for the error that I'm seeing with `aws-sdk` module, just like the one @synaestheory mentioned in the comment (https://github.com/jaredallard/nexe/issues/199#issuecomment-207029513).

This is the dumps of the error.

```
bootstrap_node.js:435
      throw new Error(`No such native module ${id}`);
      ^

Error: No such native module /Users/random/simpleapp/node_modules/aws-sdk/apis/metadata.json
    at NativeModule.require (bootstrap_node.js:435:13)
    at s (nexe.js:1:176)
    at nexe.js:1:367
    at buildServiceMap (nexe.js:271:16)
    at Object.getServices [as services] (nexe.js:300:3)
    at Object.<anonymous> (nexe.js:7089:5)
    at Object.__dirname.44../api_loader (nexe.js:7099:4)
    at s (nexe.js:1:316)
    at nexe.js:1:367
    at Object.__dirname.4../api_loader (nexe.js:414:1)
```
